addons/mail_bot/static/src/owl/components/notification_list/notification_list.js

```diff
@@ -0,0 +1,64 @@
+odoo.define('mail_bot.component.NotificationList', function (require) {
+'use strict';
+
+const NotificationList = require('mail.component.NotificationList');
+const NotificationRequest = require('mail_bot.component.NotificationRequest');
```

Comment:
`mail_bot.*` in different group as `mail.*`

Reply:
ok